btnl_client/__main__.py

```diff
@@ -160,6 +162,22 @@ def main():
             base_symbol=args.base_symbol,
         )
         print(result)
+    elif args.command == "ws-feed":
+        client = ws.BitnomialWebSocketClient("wss://bitnomial.com/exchange/ws")
+
+        channels = [
+            ws.Channel(name=ws.ChannelName.Trade, product_codes=["BUI"]),
```

Comment:
are we assuming the people using this are just going to clone this repo and they will be able to update these product codes and channels?

Reply:
I don't think they would use this `__main__.py` section except as a reference, or for interactive use, the websocket module is the exposed interface here.